roms_tools/tests/test_setup/test_boundary_forcing.py

```diff
@@ -758,3 +761,60 @@ def test_from_yaml_missing_boundary_forcing(tmp_path, use_dask):
 
         yaml_filepath = Path(yaml_filepath)
         yaml_filepath.unlink()
+
+
+@pytest.mark.stream
+@pytest.mark.skipif(
+    not _has_dask(),
+    reason="Executed only if Dask package is installed",
+)
+def test_default_glorys_dataset_loading(tiny_grid: Grid) -> None:
+    """Verify the default ERA5 dataset is loaded when a path is not provided."""
+    start_time = datetime(2010, 2, 1)
+    end_time = datetime(2010, 3, 1)
+
+    with mock.patch.dict(
+        os.environ, {"PYDEVD_WARN_EVALUATION_TIMEOUT": "90"}, clear=True
+    ):
+        bf = BoundaryForcing(
+            grid=tiny_grid,
+            source={"name": "GLORYS"},
+            type="physics",
+            start_time=start_time,
+            end_time=end_time,
+            use_dask=True,
+            bypass_validation=True,
+        )
+
+        expected_vars = {"u_south", "v_south", "temp_south", "salt_south"}
+        assert set(bf.ds.data_vars).issuperset(expected_vars)
+
+
+@pytest.mark.skipif(
+    not _has_dask(),
+    reason="Executed only if Dask package is installed",
+)
+def test_nondefault_glorys_dataset_loading(small_grid: Grid) -> None:
```

Comment:
Here we can use the `use_dask` fixture.

Reply:
`use_dask` is giving a True/False value that can be used to configure a dynamic test. In my case, this test is invalid without dask and cannot be executed.

Since it's used in other places as a source of truthiness (not a trigger for skip), I can't tweak it to do the skip.